src/TitleBindPieces/hooks/useUnboundPieces/useUnboundPieces.js

```diff
@@ -0,0 +1,72 @@
+import keyBy from 'lodash/keyBy';
+import { useQuery } from 'react-query';
+
+import {
+  useNamespace,
+  useOkapiKy,
+} from '@folio/stripes/core';
+
+import {
+  ITEMS_API,
+  LIMIT_MAX,
+  LINES_API,
+  ORDER_PIECES_API,
+  PIECE_STATUS,
+  batchRequest,
+} from '@folio/stripes-acq-components';
+
+import { getPieceStatusFromItem } from '../../../common/utils';
+
+const DEFAULT_DATA = [];
+
+export const useUnboundPieces = (titleId, poLineId, options = {}) => {
+  const { enabled = true, ...otherOptions } = options;
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace({ key: 'unbound-pieces' });
+  const filterQuery = `titleId=${titleId} and poLineId==${poLineId} and receivingStatus==${PIECE_STATUS.received} and isBound==false`;
+
+  const searchParams = {
+    limit: LIMIT_MAX,
+    query: `${filterQuery} sortby locationId`,
+  };
+
+  const {
+    data = DEFAULT_DATA,
+    isLoading,
+  } = useQuery(
```

Comment:
Colleagues from other teams have asked to pass an object in such hooks, specifying the data of the fields, to facilitate the cognitive perception of the passed arguments. I suggest following this approach.